packages/ttd-cli.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,37 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = "ttd"
+    $category = "Debuggers"
+    $toolVersion = "1-11-319-0"
```

Comment:
Using the version in an env variable break automatic updates. We removed some of these variables in https://github.com/mandiant/VM-Packages/pull/1053.

Reply:
Done.